doc/sync-label.md

```diff
@@ -0,0 +1,224 @@
+# Sync Label
+
+## Motivation
+
+I am currently using git-bug to manage my bug/feature/task list for a number
+of projects. Some of those projects have "external" bug trackers (e.g.
+my day job, or github mirrors of side projects).
+
+My personal bug repo covers 20 projects. I prefer the experience of
+interacting with a single git-bug session in `termui` versus a half-dozen
+different laggy-ass hyper-ajax web 2.0 online trackers.
+
+I would love to sync my personal bug repo with these external trackers but
+there are a couple of problems:
+
+1. Our JIRA at work has millions of bugs. I obviously don't interact with
+   all of them. I only want to sync bugs that are "relevant" to me... where
+   "relevant" can presumably be defined by matching some criteria on metadata
+   in JIRA.
+2. Our gitlab at work likewise has thousands of open pull requests. I'm not sure
+   git-bug is necessarily the best tool for doing code-review right now, but
```

Comment:
I'd like to see git-bug having a real support for code-review, but that's a ton of work. I'm not sure having just a partial view of a pull-request really make sense for a majority of user though.